### PR TITLE
Booking manager hides unfulfillable booking

### PR DESCRIPTION
**Pull request by benlovell**
Adds a button on the fulfilment view so booking manager can choose to hide /
deactivate booking requests that are considered unfulfillable after a period of
time.

These won't affect the calculation of our KPIs since they're only
calculated on the presence of an associated appointment.

screen shot 2016-09-24 at 13 44 44

### CODE REVIEW

File: app/views/appointments/new.html.erb [hunk 1]
```diff
@@ -153,6 +153,12 @@
         <%= f.button class: 'btn btn-primary btn-block t-submit' do %>
           Make appointment and notify<br><%= @appointment_form.name %>
         <% end %>
+        <%= link_to 'Hide this Booking Request',
+          @appointment_form.location_aware_booking_request,
+          method: :delete,
+          class: 'btn btn-danger btn-block t-deactivate',
+          data: { confirm: 'Are you sure?' }
```
Comment (benbarnett) [inline app/views/appointments/new.html.erb]: Shall we make this a bit more contextual? 

`Are you sure you want to hide #{@appointment_form.name}'s booking request?`

Or something?

File: app/models/user.rb [hunk 2]
```diff
@@ -20,6 +20,7 @@ class User < ActiveRecord::Base
 
   def unfulfilled_booking_requests
     booking_requests
+      .active
       .includes(:appointment)
       .where(appointments: { booking_request_id: nil })
```
Comment (AndrewVos) [inline app/models/user.rb]: This feels like business logic that could be extracted

Comment (AndrewVos) [inline app/models/user.rb]: #without_appointments or something?

Comment (benlovell) [inline app/models/user.rb]: It's already within `unfulfilled_booking_requests` which to my mind is perfectly descriptive. Does the addition of the `active` scope change this significantly?

File: app/views/appointments/new.html.erb (hunk 1 above)
Comment (benlovell) [inline app/views/appointments/new.html.erb]: This makes sense. Will do.

File: app/models/user.rb (hunk 2 above)
Comment (AndrewVos) [inline app/models/user.rb]: Wow nope I totally just didn't realise the method name! lgtm

File: app/views/appointments/new.html.erb (hunk 1 above)
Comment (benlovell) [inline app/views/appointments/new.html.erb]: screen shot 2016-09-26 at 11 07 47

Comment (benlovell) [inline app/views/appointments/new.html.erb]: @benbarnett Done.

**Pull request merged**